src/views/LimitedRestorationToFull.vue

```diff
@@ -337,6 +365,52 @@ export default class LimitedRestorationToFull extends Vue {
     } as EntitySnapshotIF
   }
 
+  /** Resource getters. */
+  get orgPersonLabel (): string {
+    return this.getResource.changeData?.orgPersonInfo.orgPersonLabel
+  }
+
+  get userEmailOptional (): boolean {
+    return this.getResource.userEmailOptional
+  }
+
+  get currentPeopleAndRoles (): Array<OrgPersonIF> {
+    return this.getOrgPeople.filter(orgPerson => !this.wasRemoved(orgPerson))
+  }
+
+  get applicantEmail (): string {
+    const currentApplicant = this.getOrgPeople.filter(orgPerson => !this.wasRemoved(orgPerson))
+    return currentApplicant[0].officer.email
+  }
+
+  /** The limited restoration state filing's approval type. */
+  get approvalType (): ApprovalTypes {
+    return this.getStateFilingRestoration?.approvalType
+  }
+
+  /** The court order draft file number. */
+  get courtOrderNumberText (): string {
+    return this.getRestoration.courtOrder?.fileNumber || ''
+  }
+
+  get isApprovedByRegistrar (): boolean {
+    return this.getStateFilingRestoration?.approvalType === ApprovalTypes.VIA_REGISTRAR
+  }
+
+  get isCourtOrderOnly (): boolean {
+    return this.getStateFilingRestoration?.approvalType === ApprovalTypes.VIA_COURT_ORDER
+  }
+
+  get showCourtOrderRadio (): boolean {
+    let courtOrderRadio
+    if (this.getStateFilingRestoration?.approvalType === ApprovalTypes.VIA_COURT_ORDER) {
+      courtOrderRadio = false
+    } else {
+      courtOrderRadio = true
+    }
+    return courtOrderRadio
```

Comment:
No need to change, but next time you could do
`const courtOrderRadio = (something !== something_else)`